support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -13,6 +13,7 @@ import { SupporterPlusThankYou } from 'pages/supporter-plus-thank-you/supporterP
 import { setUpRedux } from './setup/setUpRedux';
 import { SupporterPlusInitialLandingPage } from './twoStepPages/firstStepLanding';
 import { SupporterPlusCheckout } from './twoStepPages/secondStepCheckout';
+import { ThreeTierLandingPage } from './twoStepPages/threeTierFirstStepLanding';
```

Comment:
I know we don't do this universally, but could we keep the export and filename the same? It helps having not to guess where a component came from.